### PR TITLE
Update README to show Carthage compatibility

### PR DESCRIPTION
**Pull request by nuudles**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +19,14 @@ it, simply add the following line to your Podfile:
 pod 'BonMot'
 ```
 
+## Installation with Carthage
+
+BonMot is also compatible with [Carthage](https://github.com/Carthage/Carthage). To install it, simply add the following line to your Cartfile:
+
+```ogml
```
Comment (ZevEisenberg) [inline README.md]: This should be `ogdl` (I keep making this same mistake).

**Comment by ZevEisenberg** [comment]
@mattThousand @ateliercw I don't use Carthage. If one of you has a sec, can you confirm that this is what we need to say in order to claim Carthage compatibility?

**Comment by ateliercw** [comment]
@ZevEisenberg it matches the copy used by other carthage compatible packages

**Comment by mattThousand** [comment]
that's the gist of it, you might add a line about running `carthage update`

**Comment by ZevEisenberg** [comment]
Going to keep it minimal. We don't say how to run `pod install` either. Merging. @nuudles thanks, and congratulations on being the first non-in-house developer to land a pull request in BonMot!

**Comment by nuudles** [comment]
Thanks @ZevEisenberg ! I've got more coming too, stay tuned.

**Pull request merged**